### PR TITLE
creating functions prototype using vectors

### PR DESCRIPTION
**Pull request by sabyasachi07**

### CODE REVIEW

**Review by chandrakant100 — COMMENTED** [review]
Please use branches for pushing your changes and submitting pr.

File: helpers.h [hunk 1]
```diff
@@ -1,16 +1,19 @@
-// To be Included the header file "bmp.h"
+#include "bmp.h"
+#include <vector>
+using namespace std;
```
Comment (Himanshu40) [inline helpers.h:3]: Please remove using namespace `std` because you are using `std` everywhere even if you are including `std`.

File: helpers.h [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
+void edges(int, int, std::vector<std::vector<RGBTRIPLE>>);
 
 // Blur image
-// write protopype below this comment
+void blur(int, int, std::vector<std::vector<RGBTRIPLE>>);
 
 // Sepia image
-// write protopype below this comment
\ No newline at end of file
+void sepia(int, int, std::vector<std::vector<RGBTRIPLE>>);
```
Comment (Himanshu40) [inline helpers.h]: Overall I should say remove `using namespace std;` and keep `std` wherever you have used it.

**Review by Himanshu40 — COMMENTED** [review]
Overall please don't use `main` branch for creating PR.

**Comment by Himanshu40** [comment]
@sabyasachi07 Please close this PR after you see the comments that have been given to you.

**Comment by Himanshu40** [comment]
I am closing this PR because we are moving to test with double pointers. If it fails then we can switch to vectors and then template if possible.

**Pull request closed without merging**